packages/create/src/utils/removeEmptyProperties.ts

```diff
@@ -0,0 +1,10 @@
+export function removeEmptyProperties<T extends object>(value: T): T {
```

Comment:
The typings for this function are actually complex to get right. TODO: I'd like to make a standalone package for it.